apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -332,13 +331,6 @@ export function AppRoot(): JSX.Element | null {
     return <SetupCardReaderPage />;
   }
 
-  if (
-    stateMachineState === 'no_hardware' &&
-    !isFeatureFlagEnabled(BooleanEnvironmentVariableName.USE_MOCK_PAPER_HANDLER)
-  ) {
-    return <NoPaperHandlerPage />;
-  }
```

Comment:
Why is this screen removed?

Reply:
Agree with your conclusion, I think it's an important fail safe. To match with VxScan's handling of this case, we could use language like "Internal Connection Problem" and "Ask a poll worker for help"

Reply:
[6717124](https://github.com/votingworks/vxsuite/pull/4891/commits/6717124ec4a03a21d350afedc3c2e6fc8ef45516)